src/components/video/data/hooks.js

```diff
@@ -29,7 +33,8 @@ export function useTranscripts({ player, customOptions }) {
       }
     };
     fetchFn();
-  }, [customOptions?.transcriptUrls, player, shouldUseTranscripts]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
[clarification] Is this disabled ESLint rule still relevant (i.e., checking out the PR locally and removing the comment still doesn't throw an ESLint error)?

FWIW, this specific rule should generally be mitigated as it suggests a code smell that could result in unintended bugs during rendering.

Reply:
@adamstankiewicz I initially took a different approach and added this disable rule to prevent frequent re-renders, but it’s no longer necessary. Thanks for bringing it to my attention!